src/main/java/ro/unibuc/hello/data/product/ProductEntity.java

```diff
@@ -0,0 +1,72 @@
+package ro.unibuc.hello.data.product;
+
+import org.springframework.data.annotation.Id;
+
+public class ProductEntity {
+    @Id
```

Comment:
Class should be @Document and Id field should be @MongoId in order to have the correct _id format in mongoDB